lms/djangoapps/ccx/utils.py

```diff
@@ -260,3 +261,83 @@ def is_email(identifier):
     except ValidationError:
         return False
     return True
+
+
+def add_master_course_staff_to_ccx(course, ccx_key, display_name):
+    """
+    Added staff role on ccx to all the staff members of master course.
+
+    Arguments:
+        course (CourseDescriptorWithMixins): Master course instance
+        ccx_key (CCXLocator): CCX course key
+        display_name (str): ccx display name for email
+    """
+    list_staff = list_with_level(course, 'staff')
+    list_instructor = list_with_level(course, 'instructor')
+
+    with ccx_course(ccx_key) as course:
```

Comment:
@amir-qayyum-khan If the passed-in `course` and the `course` retrieved by `ccx_course(ccx_key)` are indeed the same, then why incur the extra modulestore call in `ccx_course()`? Why use the passed-in `course` for `list_with_level()` instead of just using the `course` gotten by `ccx_course()`? There's no explicit guarantee in this function that the passed-in `course` and the `course` retrieved by `ccx_course()` are indeed the same course.